docs/_bin/deploy-docs.sh

```diff
@@ -82,7 +82,7 @@ src=$tmp/druid
 echo "Using Version     [$version]"
 echo "Working directory [$tmp]"
 
-git clone -q --depth 1 git@github.com:apache/incubator-druid-io.github.io.git "$target"
+git clone -q --depth 1 git@github.com:apache/druid-io.github.io.git "$target"
```

Comment:
Neither the old or the new repo is valid. Is this script still being used? If not, let's delete it.

Reply:
I don't see any references to it elsewhere in the repo, and there are newer scripts for docs build/publish now, so I'll delete this